android/app/src/main/java/org/fossasia/openevent/core/auth/model/User.kt

```diff
@@ -0,0 +1,189 @@
+package org.fossasia.openevent.core.auth.model
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.github.jasminb.jsonapi.IntegerIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+import io.realm.RealmObject
+import io.realm.annotations.Ignore
+import io.realm.annotations.PrimaryKey
+
+@Type("user")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+open class User(
+        @PrimaryKey
+        @Id(IntegerIdHandler::class)
+        var id: Int = 0,
+        var firstName: String? = null,
+        var lastName: String? = null,
+        var email: String? = null,
+        @Ignore
+        var password: String? = null,
+        var isAdmin: Boolean = false,
+        var isSuperAdmin: Boolean = false,
+        var createdAt: String? = null,
+        var lastAccessedAt: String? = null,
+        var contact: String? = null,
+        var deletedAt: String? = null,
+        var details: String? = null,
+        var isVerified: Boolean = false,
+        var thumbnailImageUrl: String? = null,
+        var iconImageUrl: String? = null,
+        var smallImageUrl: String? = null,
+        var avatarUrl: String? = null,
+        var facebookUrl: String? = null,
+        var twitterUrl: String? = null,
+        var instagramUrl: String? = null,
+        var googlePlusUrl: String? = null,
+        var originalImageUrl: String? = null
+) : RealmObject() {
+
+    class UserBuilder internal constructor() {
+        private var id: Int = 0
+        private var firstName: String? = null
+        private var lastName: String? = null
+        private var email: String? = null
+        private var password: String? = null
+        private var isAdmin: Boolean = false
+        private var isSuperAdmin: Boolean = false
+        private var createdAt: String? = null
+        private var lastAccessedAt: String? = null
+        private var contact: String? = null
+        private var deletedAt: String? = null
+        private var details: String? = null
+        private var isVerified: Boolean = false
+        private var thumbnailImageUrl: String? = null
+        private var iconImageUrl: String? = null
+        private var smallImageUrl: String? = null
+        private var avatarUrl: String? = null
+        private var facebookUrl: String? = null
+        private var twitterUrl: String? = null
+        private var instagramUrl: String? = null
+        private var googlePlusUrl: String? = null
+        private var originalImageUrl: String? = null
```

Comment:
Massive duplication, not recommended. In model classes, don't create builders.

Reply:
The API before converting the class to Kotlin had a builder. Are you sure to remove the builder? The builder is used in a lot of places in the app.

Reply:
Yes, the builder is used to add 2 fields in most places. We can manually set 2 fields in those places. Not worth adding 60 lines of code